packages/modeling/src/geometries/poly3/validate.js

```diff
@@ -1,7 +1,11 @@
+const signedDistanceToPoint = require('../../maths/plane/signedDistanceToPoint')
 const vec3 = require('../../maths/vec3')
 const isA = require('./isA')
 const isConvex = require('./isConvex')
 const measureArea = require('./measureArea')
+const plane = require('./plane')
+
+const NEPS = 1e-13
```

Comment:
Maybe it's time to move this to maths/constants.js... we have multiple copies everywhere. :(

```
primitives/triangle.js:const NEPS = 1e-13
utils/trigonometry.js:const NEPS = 1e-13
maths/utils/aboutEqualNormals.js:const NEPS = 1e-13
operations/modifiers/mergePolygons.js:const NEPS = 1e-13
operations/modifiers/retessellate.js:const NEPS = 1e-13
```

Reply:
Moved NEPS into constants, and updated references to it :+1: